sources/classes/ServerConfig.cpp

```diff
@@ -0,0 +1,158 @@
+#include "../../includes/classes/ServerConfig.hpp"
+
+	/*	CONSTRUCTORS	*/
+
+ServerConfig::ServerConfig(void)
```

Comment:
Default ServerConfig should go to a #defined path. Not use code to define it. Create the default file with default configs and read it here

Reply:
The default constructor will never be called in the final version. The idea is to run the ConfigParser object on the config file to check if the syntax is valid and load the contents of the file into a vector of strings (one server block at a time). Then, each ServerConfig object will be instantiated with the correspnding string vector being passed as argument. In order to do this, if no argument is provided when running webserv, then the path for the default config file is to be used as argument for ConfigParser.
Again, the default constructor will never be used in the final version. It's currently implemented in order to 1) conform to the Orthodox Canonical Form and 2) be able to instantiate an object from this class, and it can be removed at a later time